include/drivers/gpio.h

```diff
@@ -55,12 +55,19 @@ extern "C" {
 /* Initializes output to a high state. */
 #define GPIO_OUTPUT_INIT_HIGH   (1U << 11)
 
+/* Initializes output based on logic level */
+#define GPIO_OUTPUT_INIT_LOGICAL (1U << 23)
+
 /** @endcond */
 
 /** Configures GPIO pin as output and initializes it to a low state. */
-#define GPIO_OUTPUT_LOW         (GPIO_OUTPUT | GPIO_OUTPUT_INIT_LOW)
+#define GPIO_OUTPUT_LOW      (GPIO_OUTPUT | GPIO_OUTPUT_INIT_LOW)
```

Comment:
I would keep the old alignment as it's matching other flags, unless we modify all of them.